docker/docker-run.sh

```diff
@@ -5,7 +5,14 @@ REPO_NAME="purduearc/rocket-league"
 CONTAINER_NAME="${CONTAINER_NAME:-arc-rocket-league-dev}"
 echo "mounting host directory $WS_DIR as container directory /home/$USER/catkin_ws"
 
-docker run --rm -it \
+# tty-specific options
+if [ -t 0 -a -t 1 ]
```

Comment:
Can you explain what this is doing? I assume it adds `-it` if this is an interactive terminal?

Reply:
Yeah exactly, running docker with -it throws and error if it's not running in a terminal. The action isn't in a terminal, which was causing the error that you ran into